extensions/ui-extension/src/CheckoutBlock.tsx

```diff
@@ -0,0 +1,9 @@
+import { reactExtension } from '@shopify/ui-extensions-react/checkout';
+import { PurchaseOrderInput } from './fields/PurchaseOrder';
+import { CodiceFiscale } from './fields/CodiceFiscale';
+
+export default reactExtension('purchase.checkout.block.render', () => (
+   <>
+      <PurchaseOrderInput /> <CodiceFiscale />
+   </>
+));
```

Comment:
Hm I'm worried that this won't let us render these components in independent places. We might have to split them up like:

```suggestion
export reactExtension('purchase.checkout.block.render',
   () => <PurchaseOrderInput />
);
export reactExtension('purchase.checkout.block.render',
   () => <CodiceFiscale />
);
```

Reply:
It's also probably not worth changing right now. I don't think we can test multiple dynamic zones at once, and it's not clear whether we can put more than one extension in a single dynamic zone (my hunch is no)

Reply:
> It's also probably not worth changing right now. I don't think we can test multiple dynamic zones at once, and it's not clear whether we can put more than one extension in a single dynamic zone (my hunch is no)

I'm okay with changing it in a follow-up pull when we need them rendered independently. :+1: